### ISSUE TITLE
Store calibration data after "Stop Sensor"

### ISSUE THREAD

**Issue by JoernL** [StephenBlackWasAlreadyTaken/xDrip-Experimental#249]
Sometimes it is necessary to remove the transmitter for cleaning or sensor tuning.
After re-inserting it on the same sensor, it would be nice to restore the saved calibration
data.

**Comment by tzachi-dar** [comment]
do you stop the sensor before doing that?

**Comment by JoernL** [comment]
Yes, i choose "stop sensor", remove the transmitter and then the calibrations are gone.
Removing the transmitter without stoping it and then re-insert it again cause strange values.

**Comment by tzachi-dar** [comment]
I simply know that after cleaning, I have to ignore 3-4 readings. I don't
stop the sensor for that.
Starting a sensor will also have an influence on our adjustment of bg
readings even when the same calibrations are set.

you can stop the reading, by selecting another input device, if this is
what you realy want.

**Comment by JoernL** [comment]
Ah, okay. I don't know, that it works. I will try that out.
Thanks.

**Comment by AdrianLxM** [comment]
Hi Jörn, you can simply "forget" the device in the System Status... and when you are done, you can repair to the device.
xDrip-Wixel will store no data and xBridge-Wixel just for half a minute or so. Once that time has passed, it won't send the data again.

**Comment by JoernL** [comment]
Hi Adrian, okay ... i will try this out. Thx :)
